test/service_instance_and_binding_test/service_binding_test/service_binding_test.go

```diff
@@ -224,6 +224,28 @@ var _ = DescribeTestsFor(TestCase{
 			})
 
 			Describe("GET", func() {
+
+				Context("when attach_last_operations is truthy", func() {
```

Comment:
shouldn't this be part of the generic tests for all types? it doesn't have any binding specific logic

Reply:
Fixed